hash/neogeo.xml

```diff
@@ -11493,7 +11493,7 @@ license:CC0-1.0
 		</part>
     </software>
 
-	<software name="neotris">
+	<software name="neotris" supported="partial">
```

Comment:
Please add a comment saying *why* it’s partially supported.

Reply:
Done it